### DISCUSSION TITLE
how to remove the default header

### DISCUSSION THREAD

**Discussion by np1987** [grafana/grafana-infinity-datasource#780] [Q&A]
how to remove the default header gets appeneded to the url. With that url is not accessible if we just use curl -X GET urls are working.

curl -X 'GET' **-H 'Accept: application/json;q=0.9,text/plain'** 'https://url'
curl -X 'GET' **-H 'Accept: text/xml;q=0.9,text/plain**' 'https://url'

**Comment by yesoreyeram** [comment]
You can't remove the Accept header but you can override it in the config headers section.

**Reply by np1987** [reply]
Thanks. Any examples would be helpful.

**Reply by epraetor-eolian** [reply]
Agreed - trying to update to a new version of this plugin has broken all my queries with the addition of these default headers. Please advise how to remove/override

EDIT: set the Accept and Content-Type headers to match the data for your API. (application/json for most I imagine)
